package.json

```diff
@@ -52,5 +51,9 @@
     "selector",
     "datetime",
     "ui"
-  ]
+  ],
+  "engines": {
+    "npm": "~2.13.0",
+    "node": "~0.12.0"
```

Comment:
ooh - straight into 4.0. brave! I've not been able to update yet, soon i hope!
If it works with node4 that a `>` makes sense.